app/static/src/app/store/translations/codeTab.ts

```diff
@@ -0,0 +1,16 @@
+interface CodeTabTranslations {
+    codeTabExample: string
+}
+
+const en: Partial<CodeTabTranslations> = {
```

Comment:
I don't think we should make these Partial - we do want to define values for every field in the interfaces for every language I think? If not, we should support defining a fallback language (I don't think we're doing that at the moment?).